_blocks/get-help-now/block.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Get Help Now callout
```

Comment:
[suggestion] content blocks have `name` not `title`, this change also needs to happen on the other new blocks.